projects/storefrontapp-e2e-cypress/cypress/helpers/saved-cart.ts

```diff
@@ -247,12 +247,12 @@ export function saveActiveCart(verify = true) {
       // open modal to save the cart
 
       cy.get('cx-saved-cart-form-dialog').within(() => {
-        cy.get('[formcontrolname="name"]').type(
-          sampleData.savedActiveCartForm[0].name
-        );
-        cy.get('[formcontrolname="description"]').type(
-          sampleData.savedActiveCartForm[0].description
-        );
+        cy.get('[formcontrolname="name"]')
+          .clear()
```

Comment:
During the pipeline testing process, when opening the "Save Cart" dialog, if there is already text present in the input field, it results in the typed content being appended after the existing text, leading to testing errors.